docs/docs/reference/metaprogramming/erased-terms-spec.md

```diff
@@ -3,9 +3,9 @@ layout: doc-page
 title: "Erased Terms Spec"
 ---
 
-# Implementation
+## Implementation
 
-## Rules
+### Rules
```

Comment:
Maybe remove `Implementation` and only keep `Rules`?

Reply:
Done.